test/lib/templateFactory.test.js

```diff
@@ -1249,6 +1331,164 @@ describe('Template Factory', () => {
+                        branch: 'main',
+                        name: 'test/repo',
+                        url: 'https://github.com/test/repo/tree/main/pipe1',
+                        rootDir: 'pipe1',
+                        private: false
+                    },
+                    lastRun: '2023-08-31T18:18:37.501Z',
+                    admins: { nathom: true }
+                }
+            ];
+
+            jobFactoryTestOutput = [
+                { pipelineId: 4, count: 1 },
+                { pipelineId: 5, count: 2 },
+                { pipelineId: 6, count: 1 }
+            ];
+
+            pipelineFactoryTestOutput = [
+                {
+                    id: 6,
+                    name: 'nathom/sd-uses-template',
+                    scmUri: 'github.com:672032066:main:pipe1',
+                    scmContext: 'github:github.com',
+                    scmRepo: {
+                        branch: 'main',
+                        name: 'test/repo',
+                        url: 'https://github.com/test/repo/tree/main/pipe1',
+                        rootDir: 'pipe1',
+                        private: false
+                    },
+                    createTime: '2023-08-17T18:18:37.501Z',
+                    admins: { nathom: true },
+                    lastEventId: 2
+                },
+
+                {
+                    id: 5,
+                    name: 'nathom/sd-uses-template',
+                    scmUri: 'github.com:672032066:main:pipe1',
+                    scmContext: 'github:github.com',
+                    scmRepo: {
+                        branch: 'main',
+                        name: 'test/repo',
+                        url: 'https://github.com/test/repo/tree/main/pipe1',
+                        rootDir: 'pipe1',
+                        private: false
+                    },
+                    createTime: '2023-08-17T18:18:37.501Z',
+                    admins: { nathom: true },
+                    lastEventId: null
+                },
+
+                {
+                    id: 4,
+                    name: 'nathom/sd-uses-template',
+                    scmUri: 'github.com:672032066:main:pipe1',
+                    scmContext: 'github:github.com',
+                    scmRepo: {
+                        branch: 'main',
+                        name: 'test/repo',
+                        url: 'https://github.com/test/repo/tree/main/pipe1',
+                        rootDir: 'pipe1',
+                        private: false
+                    },
+                    createTime: '2023-08-17T18:18:37.501Z',
+                    admins: { nathom: true },
+                    lastEventId: 1
+                }
+            ];
+
+            eventFactoryTestOutput = [
+                {
+                    id: 1,
+                    createTime: '2023-08-31T18:18:37.501Z'
+                },
+                {
+                    id: 2,
+                    createTime: '2023-08-17T18:18:37.501Z'
```

Comment:
Have different value for each entry
```suggestion
                    createTime: '2023-07-18T12:18:42.501Z'
```